enterprise/cmd/repo-updater/main.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/sourcegraph/sourcegraph/internal/ratelimit"
 	"github.com/sourcegraph/sourcegraph/internal/repos"
 	"github.com/sourcegraph/sourcegraph/internal/timeutil"
+	sglog "github.com/sourcegraph/sourcegraph/lib/log"
```

Comment:
There is exactly one usage in `main()` of standard library `log`, let's move that into `enterpriseInit`, use the `logger` provided, remove the standard library log, and rename this import?

```suggestion
	"github.com/sourcegraph/sourcegraph/lib/log"
```